src/pages/workspaces/workspace/Data.js

```diff
@@ -510,9 +513,18 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
           })
         ])
       ]),
+      !!selectedEntities.length && h(FloatingActionButton, {
```

Comment:
In #930, I changed the type of `selectedEntities` from an array to an object. You should be able to change this to `_.isEmpty(selectedEntities)` now and it will work with either type.